src/fugw/scripts/coarse_to_fine.py

```diff
@@ -348,6 +350,12 @@ def fit(
         Distribution weights of target nodes.
         Should sum to 1. If None, each node's weight
         will be set to 1 / m.
+    init_plan: torch.sparse_coo_tensor or None
+        Initial transport plan to use when fitting the fine mapping.
+        If None, a random plan will be used.
+    mask: torch.Tensor or None
+        Sparsity mask to use when fitting the fine mapping.
+        If None, a mask will be computed from the coarse mapping.
```

Comment:
Feel free to ignore this comment, but would rather name this variable `sparsity_mask` so that it's not confused with a nilearn masker.

Reply:
Good point